src/docs/asciidoc/installing_upgrading.adoc

```diff
@@ -0,0 +1,771 @@
+[[installing-hazelcast-imdg]]
+== Installing and Upgrading
+
+This chapter provides pre- and post-installation
+details, and deployment options of Hazelcast IMDG.
+
+=== Using Maven
```

Comment:
Not sure if maven should be the first item, especially if we're going towards non-java users. WDYT?